src/metrics/commits_per_pr/index.ts

```diff
@@ -31,25 +36,29 @@ export const collectCommitsPerPrMetrics = async (
     );
 
     const data = request.data;
-    numberOfCommits = data.length;
 
     // Checking the author date will not reflect amendments
-    commit_timestamps = data.map(({ commit }) => ({
+    const commit_timestamps = data.map(({ commit }) => ({
       authored: commit.author?.date ? getTimestamp(commit.author.date) : null,
       committed: commit.committer?.date
         ? getTimestamp(commit.committer.date)
         : null,
     }));
+
+    commits = {
+      amount: data.length,
+      commit_timestamps,
+    };
   } catch (error: unknown) {
-    // TODO: send error output
+    status = "networkError";
+    logger.error(LogErrors.networkErrorCommitsPerPR, pr_id);
```

Comment:
If there is an message in the error we should include this too in our log, could help us to identify the issue at hand.